src/tree/render.js

```diff
@@ -94,12 +95,19 @@ export function preRenderTree(element, dataTree) {
     .append('div')
     .attr('class', 'org-node-holder');
 
+  const tooltip = select(element)
+    .selectAll('.tooltip')
```

Comment:
change classname to something more specific: `sn-org-tooltip`

Reply:
Also update from master to get the css changes, and create a new file for the tooltip css/less.